DI-Policies.md

```diff
@@ -0,0 +1,51 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
```

Comment:
Is this too vague? Should I be including more specific items about *how* I would be amplifying those stories? Or is the commitment to do so enough for this particular document?

Reply:
It's a little vague for now, but you will find concrete examples to add as you learn and grow. It could be promoting D+I minded vendors in your Instagram stories or on your website, love stories from underrepresented folx...

Reply:
Listening and amplifying could also be seen as the overarching theme of this document. 🤔

Reply:
I think providing some examples of what this means could be helpful! That isn't to say you're committing to each individual example, but moreso to give your readers and clients a reference. Off the top of my head, some examples could include:

- spotlighting interracial couples and couples with URMs 
- promoting the work of URM professionals in the industry

Reply:
Thank you both so much! This is hugely helpful. I like the idea of giving a couple examples, but not necessarily committing to those exact things.